cobbler/actions/buildiso.py

```diff
@@ -677,19 +681,19 @@ def run(self, iso=None, buildisodir=None, profiles=None, systems=None, distro=No
         else:
             self.generate_netboot_iso(imagesdir, isolinuxdir, profiles, systems, exclude_dns)
 
-        if mkisofs_opts is None:
-            mkisofs_opts = ""
+        if xorrisofs_opts is None:
+            xorrisofs_opts = ""
         else:
-            mkisofs_opts = mkisofs_opts.strip()
+            xorrisofs_opts = xorrisofs_opts.strip()
 
-        # using xorrisofs as it accepts mkisofs parameters as-is and is available everywhere...
-        cmd = "xorrisofs -o %s %s -r -b isolinux/isolinux.bin -c isolinux/boot.cat" % (iso, mkisofs_opts)
+        # using xorrisofs instead of mkisofs nowadays, it is available everywhere...
```

Comment:
I think the comma is in the wrong spot! :wink: 
```suggestion
        # using xorrisofs instead of mkisofs, nowadays it is available everywhere...
```